edge-apps/iframe/static/js/main.js

```diff
@@ -0,0 +1,20 @@
+/* global  screenly */
+
+function extractUrlFromIframeSetting (settingValue) {
```

Comment:
We might not need this if we just take the URL as the setting value

Reply:
Agree, but we need to support both URL and full iFrame tag.